documentation/docs/connectors/gate-io.md

```diff
@@ -21,6 +21,7 @@ To use [Gate.io](https://www.gate.io/ref/4566709) connector for spot trading wit
 
 ![](/assets/img/gateio-account.png)
 
+
```

Comment:
Unnecessary line space.

Reply:
remove unnecessary line with the latest commit